src/design-tokens/mixins.css

```diff
@@ -9,11 +9,11 @@
 @define-mixin textLink {
   @mixin eds-theme-typography-body-text-md;
   text-decoration: underline;
-  color: var(--eds-theme-color-text-link-strong);
+  color: var(--eds-theme-color-text-link);
```

Comment:
I changed this one back to the default tokens. I feel like this mixin is kind of pointless now because our components are set up in such a way that you can just use one of those, but I'll leave this for now in case I'm missing something.

Reply:
I think I agree with `--eds-theme-color-text-link` as far as nomenclature goes and if the underlying tier 1 token itself changes, this should stay the same